gui/wxpython/nviz/mapwindow.py

```diff
@@ -382,10 +382,7 @@ def OnEraseBackground(self, event):
 
     def OnSize(self, event):
         size = self.GetClientSize()
-        if CheckWxVersion(version=[2, 9]):
-            context = self.context
-        else:
-            context = self.GetContext()
+        context = self.context if CheckWxVersion(version=[2, 9]) else self.GetContext()
```

Comment:
_True if current wx version is greater or equal than specified version otherwise False_

```suggestion
        context = self.context
```
But the check is in several places, so perhaps there is no point in removing it here only.

Reply:
Ok, so let me keep it for the follow up PR(s) that I plan to do that builds upon this series of 3 PRs. (Problems we found in code, but not related to the changes specifically).

I plan to work on it somewhere this weekend, so I hope to have the open PRs merged to be able to advance without having to redo the work because of conflicts